be/src/exec/vectorized/aggregate/aggregate_blocking_node.cpp

```diff
@@ -27,6 +28,10 @@ Status AggregateBlockingNode::open(RuntimeState* state) {
 
     VLOG_ROW << "group_by_expr_ctxs size " << _aggregator->group_by_expr_ctxs().size() << " _needs_finalize "
              << _aggregator->needs_finalize();
+    bool group_by_with_limit = (!_aggregator->is_none_group_by_exprs() &&     // has group by
```

Comment:
The name of `group_by_with_limit` would better expression the aggregate functions is not empty